### PR TITLE
Bug 986960 - Intermittent erroring test, fullscreen_image_test.js

### PR DESCRIPTION
**Pull request by nojunpark**
fixing the script so it matches the current gallery app behavior

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=477b34c445fdc5448993e4685afe3fecd12ae7e4

File: apps/gallery/test/marionette/fullscreen_image_test.js [hunk 1]
```diff
@@ -49,19 +49,38 @@ marionette('the gallery', function() {
     assert.ok(app.thumbnailsView.displayed());
   });
 
-  test.skip('should flick through images in fullscreen mode', function() {
+  test('should flick through images in fullscreen mode', function() {
```
Comment (punamdahiya) [inline apps/gallery/test/marionette/fullscreen_image_test.js]: nit: As we are moving away from webgl, we don't need webgl.force-enabled permission and can be removed from Line 17

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2908c39bc90b7dc0906a15b5888624c00da142eb

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1cdf7803cd1977cf0f9676e61711685c6f45082b

**Pull request merged**